.editorconfig

```diff
@@ -0,0 +1,31 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{py,rst,ini}]
+indent_style = space
+indent_size = 4
```

Comment:
reST blocks should use 3-space indents (and code inside reST code blocks 4-space, of course)

Reply:
Oups, I copied this from another project. Since they are not used here I could just remove the `rst` extension here.